util/src/test/java/tech/pegasys/teku/util/config/KeyStoreFilesLocatorTest.java

```diff
@@ -201,7 +199,7 @@ public void shouldHandleSymlinkedDirectories(@TempDir final Path tempDir) throws
     try {
       Files.createSymbolicLink(tempDir.resolve("key"), realKeyDir);
       Files.createSymbolicLink(tempDir.resolve("pass"), realPassDir);
-    } catch (UnsupportedOperationException e) {
+    } catch (IOException e) {
```

Comment:
hmm, the JavaDoc explicitly says it will throw `UnsupportedOperationException` if symlinks aren't supported.  Probably should catch both exception types to be safe.

Reply:
`FileSystemException` is thrown when not enough privilege on Win (which is usually the case). I think we could better just disable this case for Windows